src/bosh_azure_cpi/spec/manual_tests/performance/tf/main.tf

```diff
@@ -0,0 +1,4 @@
+module "namings" {
```

Comment:
Is it possible to create a test environment using https://github.com/Azure/azure-quickstart-templates/tree/master/bosh-setup instead of terraform?

Reply:
in this test, no bosh environment is needed.